src/demos/select-field/select-field-demo.component.html

```diff
@@ -142,3 +142,44 @@ <h2 class="sky-section-heading">
     </div>
   </sky-column>
 </sky-row>
+
+<sky-row>
+  <sky-column screenSmall="12">
+    <h2 class="sky-section-heading">
+      Custom picker
+    </h2>
+
+    <div class="sky-form-group">
+      <label
+        class="sky-control-label"
+        id="my-select-field-custom-1">
+        What is your favorite color?
+      </label>
+      <sky-select-field
+        [data]="colorStream"
+        ariaLabelledBy="my-select-field-custom-1"
+        selectMode="single"
+        singleSelectClearButtonTitle="Clear selected color"
+        singleSelectOpenButtonTitle="Select color"
+        singleSelectPlaceholderText="Select color..."
+        name="favoriteColorCustom"
+        [customPicker]="customFavoriteColorPicker">
```

Comment:
Code style > check your ordering of inputs here.
https://host.nxt.blackbaud.com/skyux-team-docs/style#component-html-tags